src/api/subscriptions.js

```diff
@@ -206,9 +206,10 @@ export let OffersRequest;
 export let LoginRequest;
 
 /**
- * @typedef {{
- *   token: string,
- * }}
+ * Properties:
+ * - one and only one of "token" or "authCode" and
+ * - corresponding value
+ * @const {Object<string, string>}
```

Comment:
This should continue to be typedef. It should look like this:

```
@typedef {{
  token: (string|undefined),
  authCode: (string|undefined),
}}
```

Reply:
can't have both @const {Object<string, string>} and @typedef right? I will keep the typedef because it's more accurate and comment that only one of they entries can exist.

Reply:
No. But it's definitely not a `@const` and `Object<string, string>` is not a great fit here.